bHY6yh-d46317/cod-pol7d7_bHY6yh-d46317.xml

```diff
@@ -0,0 +1,150 @@
+               <name ref="#jeffreycwitt">Jeffrey C. Witt</name>
+               <resp>Transcription Editor</resp>
+               <resp>TEI Encoder</resp>
+            </respStmt>
+            <respStmt>
+            <name>Katrina Van Der Ahe</name>
+            <resp>Transcription Editor</resp>
+            <resp>TEI Encoder</resp>
+            </respStmt>
+        </titleStmt>    
+         <editionStmt>
+            <edition n="0.0.0-dev">
+               <title>Rubrica 2</title>
+               <date when="2023-06-21">June 21, 2023</date>
+            </edition>
+         </editionStmt>
+         <publicationStmt>
+            <authority>SCTA</authority>
+            <availability status="free">
+               <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+               <witness xml:id="K" n="cod-pol7d7">Krakow, Biblioteka Jagiellonska MS lat. 700 III (CC.VIII.25)</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2021-05-18" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#K" n="101-r"/> <!--double check page number-->
+            <cb ed="#K" n="a"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-d46317"><!-- p3l1-r2 -->
+            <head xml:id="bHY6yh-d46317-Hd1e108">Rubrica 2</head>
+            <head xml:id="bHY6yh-d46317-Hd1e111" type="question-title">De experientia</head>
+            <p xml:id="bHY6yh-d46317-d1e114">
+              De exp<lb ed="#K" break="no"/>erientia. 
+              Rubrica. e2
+            </p>
+            <p xml:id="bHY6yh-d46317-d1e119">
+              <lb ed="#K"/> <ref xml:id="bHY6yh-d46317-Rd1e115">ARistoteles ethicorum li. i. c. i.</ref> <quote xml:id="bHY6yh-d46317-Qd1e127">Iu<lb ed="#K" break="no"/>uenis 
+              inexpertus est eorum qui secundum 
+              <lb ed="#K"/>vitam sunt actuum</quote>
+            </p>
+            <p xml:id="bHY6yh-d46317-d1e133">
+              <ref xml:id="bHY6yh-d46317-Rd1e130">li. 6. c. 5.</ref> <quote xml:id="bHY6yh-d46317-Qd1e135">Iuue<lb ed="#K" break="no"/>nis
```

Comment:
I think we should start expanding obvious reference abbreviations. 

So "l." should be "libro" and "c." should be "capitulo" and "fi." should be "fine"

(No need to correct previous transcriptions; but I would start expanding these going forward. 

(if the abbreviations is very uncertain, then you can leave as is.)

Reply:
I will make those additions when I go through and do my textual edit of the text this version is just formatting so I can take multiple read throughs of the text to ensure accuracy of my transcription